bff/cmd/main.go

```diff
@@ -55,9 +56,20 @@ func run(webappPath string) error {
 	var config Config
 	config.WebappPath = webappPath
 	config.Port = rawConfig["port"].(string)
+
+	// Get access key from environment variable with fallback to config file
+	accessKey := os.Getenv("ACCESS_KEY")
```

Comment:
I'd suggest making the ENV var name the same as the header name to reduce confusion. So, change this to "AUTODBA_ACCESS_KEY"

Reply:
Prefix-based namespacing of environment variables makes sense.

Reply:
Done.